src/main/PlagueMain.java

```diff
@@ -497,12 +499,12 @@ public void init() {
                     return;
                 }
                 PlagueTeam team = teams.get(event.unit.team);
-                team.monos = min((byte) (team.monos + 1), (byte) 255);
+                team.monos = (short) Math.min((int) (team.monos + 1), (int) MONO_LIMIT);
```

Comment:
at this point, why not just make mono limit and monos ints

Reply:
no

Reply:
fine